subprojects/docs/src/main/resources/header.html

```diff
@@ -91,7 +91,6 @@
         <h3 id="introduction">Introduction</h3>
         <ul>
             <li><a href="../userguide/what_is_gradle.html">What is Gradle?</a></li>
-            <li><a href="https://gradle.com/">Gradle Enterprise</a></li>
```

Comment:
Hmmm I think this is fine to remove as its not documentation - however other stakeholders may feel differently.

Reply:
I discussed this with @pioterj in our 1:1. We don't think GE should have a menu item in the manual but it should be mentioned.